src/unix/linux_like/emscripten/align.rs

```diff
@@ -37,10 +37,12 @@ macro_rules! expand_align {
                 size: [u8; ::__SIZEOF_PTHREAD_COND_T],
             }
 
+            // Lowered from 16 to 8 bytes in
+            // https://github.com/llvm/llvm-project/commit/d1a96e906cc03a95cfd41a1f22bdda92651250c7
```

Comment:
This can just go in the commit message

Reply:
Split into commit 52fa133223a5a338494f1abab42e2a47a464e0d6 and moved the `time_t` description to the commit message as well.
https://github.com/rust-lang/libc/compare/b6a957f2ec9f24a843d04ec4313f3a4634944072..a37fa9d267ddb3c7cbc9504ac730a7ed6fc11f59